ForumProject/ForumProject/settings.py

```diff
@@ -171,7 +175,60 @@
     },
 }
 
+
+REST_FRAMEWORK = {
+    'DEFAULT_PERMISSION_CLASSES': [
+        'rest_framework.permissions.AllowAny',
+    ],
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        # 'rest_framework.authentication.BasicAuthentication',
+        # 'rest_framework.authentication.SessionAuthentication',
+        'rest_framework_simplejwt.authentication.JWTAuthentication',
```

Comment:
Consider, that `AllowAny` is insecure for production as it will allow unrestricted access to your API

Reply:
Done, thanks